crates/sui-bridge/src/config.rs

```diff
@@ -117,12 +117,23 @@ pub struct BridgeNodeConfig {
     /// Network key used for metrics pushing
     #[serde(default = "default_ed25519_key_pair")]
     pub metrics_key_pair: NetworkKeyPair,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub metrics: Option<MetricsConfig>,
```

Comment:
let's call this `metrics_config`

Reply:
`metrics` is what it is called in `sui-node.yaml` validator configs. Probably better to keep them the same?